packages/angular/cli/commands/run.json

```diff
@@ -1,8 +1,8 @@
 {
   "$schema": "http://json-schema.org/schema",
   "$id": "ng-cli://commands/run.json",
-  "description": "Runs a custom target defined in your project.",
-  "$longDescription": "",
+  "description": "Runs an architect target with an optional custom builder configuration defined in your project.",
```

Comment:
Edit: Architect in caps everywhere? (I don't have a pref. This is the only lowercase i noticed so far)